packages/page-accounts/src/modals/Create.tsx

```diff
@@ -262,147 +278,172 @@ function Create ({ className = '', onClose, onStatusChange, seed: propsSeed, typ
-            />
-            <ExternalWarning />
-            <Expander
-              className='accounts--Creator-advanced'
-              isPadded
-              summary={t<string>('Advanced creation options')}
-            >
-              <Modal.Columns>
-                <Modal.Column>
-                  <Dropdown
-                    defaultValue={pairType}
-                    help={t<string>('Determines what cryptography will be used to create this account. Note that to validate on Polkadot, the session account must use "ed25519".')}
-                    label={t<string>('keypair crypto type')}
-                    onChange={_onChangePairType}
-                    options={uiSettings.availableCryptos}
-                    tabIndex={-1}
-                  />
-                </Modal.Column>
-                <Modal.Column>
-                  <p>{t<string>('If you are moving accounts between applications, ensure that you use the correct type.')}</p>
-                </Modal.Column>
-              </Modal.Columns>
-              <Modal.Columns>
-                <Modal.Column>
-                  <Input
-                    help={t<string>('You can set a custom derivation path for this account using the following syntax "/<soft-key>//<hard-key>". The "/<soft-key>" and "//<hard-key>" may be repeated and mixed`. An optional "///<password>" can be used with a mnemonic seed, and may only be specified once.')}
-                    isError={!!deriveValidation?.error}
-                    label={t<string>('secret derivation path')}
-                    onChange={_onChangeDerive}
-                    onEnter={_onCommit}
-                    placeholder={
-                      seedType === 'raw'
-                        ? pairType === 'sr25519'
-                          ? t<string>('//hard/soft')
-                          : t<string>('//hard')
-                        : pairType === 'sr25519'
-                          ? t<string>('//hard/soft///password')
-                          : t<string>('//hard///password')
-                    }
-                    tabIndex={-1}
-                    value={derivePath}
-                  />
-                  {deriveValidation?.error && (
-                    <article className='error'>{errorIndex[deriveValidation.error] || deriveValidation.error}</article>
-                  )}
-                  {deriveValidation?.warning && (
-                    <article className='warning'>{errorIndex[deriveValidation.warning]}</article>
-                  )}
-                </Modal.Column>
-                <Modal.Column>
-                  <p>{t<string>('The derivation path allows you to create different accounts from the same base mnemonic.')}</p>
-                </Modal.Column>
-              </Modal.Columns>
-            </Expander>
-          </Modal.Content>
-        )
-      }
+          </Expander>
+          <Checkbox
+            label={<>{t<string>('I have saved my mnemonic seed safely')}</>}
```

Comment:
This checkbox doesn't "pop" - not your fault, since it applies label, but it is not quite obvious enough.

Reply:
You mean doesn't catch user's attention visually? Or do you mean the styling when selected?

Reply:
It is not obvious that it needs to be checked at all. I even had to look around before figuring out - the actions should be more in the face, that component certainly doesn't do it. Possibly -

- centering it (it is not consistent, but would allow for more visual draw possibly?)
- overriding the color on that one to be "normal"

So both are inconsistent, however unless we come up with something else, cannot see how to just draw the user's attention to actually making an action.